### PR TITLE
Simplify installation instructions in README

### PR DESCRIPTION
**Pull request by chrisyeh96**
Simplify installation instructions in README

Also link to the latest Python 3 documentation for pip, instead of the Python 3.6 documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,34 +45,23 @@ Read the current JupyterLab documentation on [ReadTheDocs](http://jupyterlab.rea
...
-### pip
 
-If you use `pip`, you can install it with:
+# mamba
+mamba install -c conda-forge jupyterlab
 
-```shell
+# pip
 pip install jupyterlab
 ```
```
Comment (krassowski) [inline README.md:59]: I am afraid that novice users (which are a sizeable portion of the userbase, e.g. non-CS students learning statistics) will copy-paste all three commands. I find this surprisingly common, but I understand why this happens. As running both conda and pip in a single environment can cause issues, I would prefer to keep them separate with explicit "If you use X then do X" as it requires users to *think* about which option is appropriate for them.

I am fine with movin the "For more detailed instructions" paragraph down though.

Comment (chrisyeh96) [reply README.md]: Good point. I have fixed this with a new commit. Let me know what you think!

**Review by krassowski — APPROVED** [review]

**Comment by blink1073** [comment]
Hi @chrisyeh96, can you please run `jlpm run integrity` locally and push those changes?

**Comment by chrisyeh96** [comment]
Hi @blink1073, I only modified a README.md file. I did not modify any code. To me, it seems overkill to run a package integrity check for a README file change. However, I can run it if you insist. Just let me know!

**Comment by blink1073** [comment]
Yes, please.  The readme is part of the integrity script, and is why the the checks are failing.

**Pull request merged**